### PR TITLE
Add some more web browser mimicry

### PR DESCRIPTION
**Pull request by cyberpower678**
Mimic a Firefox Web browser request.  Some sites are incredibly stingy
on proper requests.

Added a stingy URL that’s alive to test the changes.

### CODE REVIEW

File: src/CheckIfDead.php [hunk 1]
```diff
@@ -271,8 +298,8 @@ protected function processCurlResults( $curlInfo, $full = false ) {
 		}
 		// Check for error messages in redirected URL string
 		if ( strpos( $effectiveUrlClean, '404.htm' ) !== false ||
-			strpos( $effectiveUrlClean, '/404/' ) !== false ||
-			stripos( $effectiveUrlClean, 'notfound' ) !== false
+		     strpos( $effectiveUrlClean, '/404/' ) !== false ||
+		     stripos( $effectiveUrlClean, 'notfound' ) !== false
```
Comment (Niharika29) [inline src/CheckIfDead.php]: Seems like you replaced tabs with spaces here?

Comment (cyberpower678) [inline src/CheckIfDead.php]: PHPStorm did.  It's an alignment thing for readability.

File: src/CheckIfDead.php [hunk 2]
```diff
@@ -48,19 +48,22 @@ class CheckIfDead {
 	 * Check if a single URL is dead by performing a full curl request
 	 *
 	 * @param string $url URL to check
+	 *
```
Comment (cyberpower678) [inline src/CheckIfDead.php]: Really?  I can remove them, but sounds a bit unusual.

File: src/CheckIfDead.php (hunk 1 above)
Comment (Niharika29) [inline src/CheckIfDead.php]: Lets just stick with tabs everywhere. :)

File: src/CheckIfDead.php (hunk 2 above)
Comment (Niharika29) [inline src/CheckIfDead.php]: Nope, it's pretty standard.

File: src/CheckIfDead.php (hunk 1 above)
Comment (cyberpower678) [inline src/CheckIfDead.php]: Have you checked the file itself.  It looks pretty nice, but I can change it back too.

File: src/CheckIfDead.php [hunk 3]
```diff
@@ -130,68 +131,8 @@ public function areLinksDead( $urls ) {
...
-	}
 
-	/**
-	 * Perform a complete text request, not just for headers
-	 *
-	 * @param array $urls URLs we are checking
-	 * @return array with params 'error':curl error number and
-	 *   'result':true(dead)/false(alive) for each element
-	 */
-	protected function performFullRequest( $urls ) {
```
Comment (Niharika29) [inline src/CheckIfDead.php]: Why are you moving these functions? 
I'd really like to only see relevant changes in this pull request. If you want to do formatting changes, they should be in a different PR.

**Comment by cyberpower678** [comment]
I'm not sure why PHPStorm moved those functions.

Gesendet von Maximilian's iPhone. 
(Sent from Maximilian's iPhone.)
